test/parallel/test-fs-read-stream-inherit.js

```diff
@@ -52,7 +52,7 @@ file.on('close', function() {
   //assert.equal(fs.readFileSync(fn), fileContent);
 });
 
-var file3 = fs.createReadStream(fn, Object.create({encoding: 'utf8'}));
+var file3 = fs.createReadStream(fn, {encoding: 'utf8'});
```

Comment:
@Fishrock123 If the `Object.create` is not removed, we cannot make a copy with `util._extend`, as it uses `Object.keys` to find the keys to be copied.